### PR TITLE
[Xamarin.Android.Build.Tests] Fix the Tls tests

### PR DESCRIPTION
**Pull request by dellis1972**
This commit fixes the Tls tests. The first problem was that
the Ionic.Zip dll was not being copied local..

The second was the path to the apk needed to use the
Root property. This property provides the root path to the
unit test working directory.

### CODE REVIEW

**Comment by dellis1972** [comment]
Weird. the failing test works fine locally. Is there any way to get the artifacts from Jenkins?

**Comment by jonpryor** [comment]
1. Should this be fixing the `Xamarin.Android.Build.Tests.BuildTest.BuildWithTlsProvider("btls",True,True)` test?

2. Why is Ionic.Zip being used *at all*? I thought we had removed that.

**Comment by jonpryor** [comment]
@dellis1972: 

> Is there any way to get the artifacts from Jenkins?

Yes.

1. Go to: https://jenkins.mono-project.com/view/Xamarin.Android/job/xamarin-android-pr-builder/
2. Click **log in** in the upper-right corner. You may need to hit #release-engineering to login.
3. Click the [**Workspace**](https://jenkins.mono-project.com/view/Xamarin.Android/job/xamarin-android-pr-builder/ws/) link in the left-hand column.

You are then able to explore the build tree.

**Comment by dellis1972** [comment]
@jonpryor woot green build!

**Comment by jonpryor** [comment]
It's "green" in part because I updated Jenkins to ignore a count of 5 errors. :-)

**Comment by dellis1972** [comment]
:(

File: src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Xamarin.ProjectTools.csproj [hunk 1]
```diff
@@ -109,7 +105,15 @@
...
   </ItemGroup>
   <ItemGroup>
-    <None Include="Ionic.Zip.xml" />
     <None Include="packages.config" />
+    <Content Include="../../../../bin/$(Configuration)/lib/xbuild/Xamarin/Android/libzip.3.0.dylib">
+        <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\..\external\LibZipSharp\libZipSharp.csproj">
```
Comment (jonpryor) [inline src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Xamarin.ProjectTools.csproj]: This should use `$(LibZipSharpSourceFullPath)`.

File: src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Xamarin.Android.Build.Tests.csproj [hunk 2]
```diff
@@ -53,6 +49,10 @@
       <Project>{2DD1EE75-6D8D-4653-A800-0A24367F7F38}</Project>
       <Name>Xamarin.ProjectTools</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\..\..\external\LibZipSharp\libZipSharp.csproj">
```
Comment (jonpryor) [inline src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Xamarin.Android.Build.Tests.csproj]: This should use `$(LibZipSharpSourceFullPath)`.

**Comment by dellis1972** [comment]
@jonpryor found the test problem and fixed it. All the msbuild unit tests now pass.

**Comment by dellis1972** [comment]
@jonpryor good to go :)

**Pull request merged**